package/base-files/files/lib/functions/system.sh

```diff
@@ -174,6 +174,18 @@ macaddr_setbit_la() {
 	macaddr_setbit $1 7
 }
 
+macaddr_unsetbit_mc() {
```

Comment:
I'd consider bumping PKG_RELEASE for base-files here, as what you add can be considered a feature ...

Reply:
Done.